prisma/schema.prisma

```diff
@@ -91,9 +93,9 @@ model UserTeam {
 }
 
 model VerificationToken {
-  identifier String
-  token      String   @unique
-  expires    DateTime
+  identifier  String   //DataroomId if dataroom else documentId
+  token       String   @unique
+  expiresAt   DateTime @default("2100-01-01T00:00:00.000Z")
```

Comment:
I think this is not a good practice to put a default value for the expiration time here in the Schema file.
`expiresAt     DateTime` this should be like this
You can handle this expiration logic where the Token Verification is done! 

@digant2482

Reply:
This was something I was testing. Didn't need this after all. Thank you for pointing, I will remove  this.